### PR TITLE
Fix/r4 signalr

### PR DESCRIPTION
**Pull request by losolio**

### CODE REVIEW

File: src/Spark.Web/Hubs/MaintenanceHub.cs [hunk 1]
```diff
@@ -17,155 +17,141 @@
 
 namespace Spark.Web.Hubs
 {
-    public class MaintenanceHub : Hub
+  //[Authorize(Policy = "RequireAdministratorRole")]
```
Comment (losolio) [inline src/Spark.Web/Hubs/MaintenanceHub.cs:20]: @kennethmyhra was this intended?

File: src/Spark.Web/Hubs/MaintenanceHub.cs [hunk 2]
```diff
@@ -17,155 +17,141 @@
...
+      {
+        await _hubContext.Clients.All.SendAsync("UpdateProgress", "Starting clearing database...");
+        await _fhirStoreAdministration.CleanAsync();
+
+        await _hubContext.Clients.All.SendAsync("UpdateProgress", "... and cleaning indexes...");
+        await _fhirIndex.CleanAsync();
+        await _hubContext.Clients.All.SendAsync("UpdateProgress", "Database cleared");
+      }
+      catch (Exception e)
+      {
```
Comment (kennethmyhra) [inline src/Spark.Web/Hubs/MaintenanceHub.cs]: Can you log the exception here, same as on line 105

File: src/Spark.Web/Hubs/MaintenanceHub.cs [hunk 3]
```diff
@@ -17,155 +17,141 @@
...
             }
-            catch (Exception e)
+            else
             {
-                await notifier.Progress("Error: " + e.Message);
+              await _fhirService.CreateAsync(key, res);
             }
+          }
+          catch (Exception e)
+          {
```
Comment (kennethmyhra) [inline src/Spark.Web/Hubs/MaintenanceHub.cs]: Log exception here as well

File: src/Spark.Web/Hubs/MaintenanceHub.cs [hunk 4]
```diff
@@ -17,155 +17,141 @@
...
+            var msgError = $"ERROR Importing {res.TypeName.ToString()}, id {res.Id}...";
+            await _hubContext.Clients.All.SendAsync("UpdateProgress", msgError);
+          }
         }
+
+        await _hubContext.Clients.All.SendAsync("UpdateProgress", "Finished loading examples");
+      }
+      catch (Exception e)
+      {
```
Comment (kennethmyhra) [inline src/Spark.Web/Hubs/MaintenanceHub.cs]: Log exception here as well

**Review by kennethmyhra — CHANGES_REQUESTED** [review]

**Pull request merged**